docs/docs/Orca/orca-pytorch-estimator.md

```diff
@@ -46,7 +46,7 @@ Estimator.from_torch(*,
 * `scheduler_creator`: parameter for `horovod` and `ray` backends. a learning rate scheduler wrapping the optimizer. You will need to set ``scheduler_step_freq="epoch"`` for the scheduler to be incremented correctly.
 * `training_operator_cls`: parameter for `horovod` and `ray` backends. Custom training operator class that subclasses the TrainingOperator class. This class will be copied onto all remote workers and used to specify custom training and validation operations. Defaults to TrainingOperator.
 * `initialization_hook`: parameter for `horovod` and `ray` backends.
-* `config`: parameter for `horovod` and `ray` backends. Config dict to create model, optimizer loss and data.
+* `config`: parameter for `horovod` and `ray` backends. Config dict, CfgNode or any class plays a role of configuration to create model, optimizer loss and data.
```

Comment:
just Config object?

Reply:
> just Config object?

I think for now arg `config` only take charge of containing config options, it could be dict/CfgNode/any simple even customized class